workspaces/src/result.rs

```diff
@@ -81,6 +81,13 @@ impl CallExecutionDetails {
         borsh::BorshDeserialize::try_from_slice(&buf).map_err(Into::into)
     }
 
+    /// Grab the underlying raw bytes returned from calling into a contract's function.
+    /// If we want to deserialize these bytes into a rust datatype, use [`CallExecutionDetails::json`]
+    /// or [`CallExecutionDetails::borsh`] instead.
+    pub fn raw_bytes(&self) -> anyhow::Result<Vec<u8>> {
```

Comment:
I wonder if raw_bytes could be ambiguous with the raw response bytes rather than just the return bytes. Maybe another opinion on this naming might be worthwhile for something like `return_bytes`

Reply:
@volovyk-s @itegulov what do you think would be better naming here for grabbing a contract's executed function return value as bytes?

Reply:
I read the current API for the other two calls as `d.as_json()`.  So `bytes` would fit the pattern.

Reply:
The problem with `return_bytes` is that it reads as a verb + noun while it is actually adj + noun. I think `raw_bytes` makes sense to me

Reply:
Yeah true, but it was intended as an adjective :D as in these are the "return value" bytes, but yeah I see the confusion here

Reply:
Alright, guess that's settled. `raw_bytes` it is then